.github/workflows/release.yml

```diff
@@ -66,6 +66,17 @@ jobs:
           gpg-connect-agent "RELOADAGENT" /bye
           gpg-connect-agent "PRESET_PASSPHRASE ${keygrip} -1 ${hexPassphrase}" /bye
           gpg-connect-agent "KEYINFO ${keygrip}" /bye
+      # Prevents dev dependencies from being reported in SBOM
+      - name: Prepare for SBOM
+        run: rm ./package-lock.json ./tests/lib/screenshot-testing/package-lock.json
+      - name: Create SBOM
+        run: |
+          mkdir -p "$HOME/.local/bin"
+          echo "$HOME/.local/bin" >> $GITHUB_PATH
+          export bin=docker-scout_1.6.3_linux_$(dpkg --print-architecture).tar.gz
+          curl -OL https://github.com/docker/scout-cli/releases/download/v1.6.3/$bin
+          tar -xvzf $bin -C "$HOME/.local/bin"
```

Comment:
There's https://github.com/docker/scout-action CI action, we don't need to download and unpack the binary.

Reply:
There was a piece m issing that is being added now. You also would need a Scout account. If you have env variable for the Scout account, I can convert it to use the action.

Reply:
@sgiehl should we get a free subscription of [Docker Scout](https://docs.docker.com/subscription/scout-details/)? It's free for up to 3 repositories. Or should we just roll with installing the binary as suggested here?

Reply:
Good questions. I guess as it only happens on release the speed factor isn't much relevant, so either way would be fine.

Reply:
I slightly prefer the Github Action as it's maintained by someone else and we can set it and forget it, compared to installing a specific version of the binary that may become outdated at some point and unless we have a reminder task or it stops working we won't look at it for another 2 years. On the other hand the binary approach is done and can be tested whereas with the account we'd need to go through a whole process to get it going.

Good consideration regarding the speed, I agree that for a release action once in a while it's ok to add a bit of time to the action runtime.

I'd say let's go with the current approach here, for now.